spec/models/pfmp_spec.rb

```diff
@@ -100,7 +100,9 @@
     end
 
     it "takes into account the previous payments" do
-      create(:pfmp, :paid, schooling: schooling)
+      max = mef.wage.yearly_cap
+
+      create(:payment, :successful, amount: max - 10, pfmp: pfmp)
```

Comment:
```suggestion
      yearly_cap = mef.wage.yearly_cap

      create(:payment, :successful, amount: yearly_cap - 10, pfmp: pfmp)
```

Le naming de l'attribut est bon et plus explicite.